carpics_360v0.1.js

```diff
@@ -710,6 +1302,81 @@ var CarPicsSpinnerAPI = (function() {
     var callback =function() {
         var spinners = [];
         var CarpicsDivs = document.getElementsByClassName("carPicsSpinner");
+        var CarpicsDiv = CarpicsDivs[0];
```

Comment:
This should be in the CarPicsSpin constructor.  Every CarpicsDiv needs these elements.